src/history/HistoryTestsUtils.h

```diff
@@ -22,7 +29,16 @@ class HistoryManager;
 namespace historytestutils
 {
 
+enum class ArchiveFileState
```

Comment:
`ArchiveFileState` is not a good name for this. Call it `TestBucketFileType` that gives a better idea of what it is